### PR TITLE
[[ Bug 9571 ]] Document menu item tags

### PR DESCRIPTION
**Pull request by montegoulding**

### CODE REVIEW

**Comment by montegoulding** [comment]
Note I've accidentally named the branch incorrectly however the note and commit message are correctly named

File: docs/dictionary/message/menuPick.lcdoc [hunk 1]
```diff
@@ -68,6 +68,39 @@ menuHistory <property>.
...
+
+    <label> [ '/' [ <accelerator> ] [ '|' <tag> ] ]
+    
+Note that the <tag> is optional as is the <accelerator> however, if you
+want an item with a tag but without the accelerator you need:
+
+    <label> '/' '|' <tag>
+  
+The <tag> must only be composed of characters from the ASCII character
+set.
```
Comment (livecodeali) [inline docs/dictionary/message/menuPick.lcdoc]: Is this still true?

Comment (montegoulding) [reply docs/dictionary/message/menuPick.lcdoc]: Hmm... that's a good question! I suspect not but I'll need to test to work it out.

Comment (montegoulding) [reply docs/dictionary/message/menuPick.lcdoc]: I tested this an in a pulldown button the following works as you would expect:

```
set the text of btn 1 to ("Bear/|ʕ•ᴥ•ʔ" & return & "Table Flip/|(ノಠ益ಠ)ノ彡┻━┻ ")
```

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 4954a1e

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 4954a1ea05620c5f734c574abfa15b1f1fb19f0b

**Comment by livecode-vulcan** [comment]
😎 test success 4954a1ea05620c5f734c574abfa15b1f1fb19f0b
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/122)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/122)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/122)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/122)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/122)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/122)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/122)

**Pull request merged**